### PR TITLE
Jump host and connection timeout

### LINKED ISSUE #65: Make use of a jump server for connecting LPARs in private network
Allow user to use a jump server (bastion_host in terms of Terraform) to allow running TF remotely for a private network cluster.

### PR DESCRIPTION
**Pull request by yussufsh**
Fixes #65

Signed-off-by: Yussuf Shaikh <yussuf@us.ibm.com>

### CODE REVIEW

File: modules/1_bastion/bastion.tf [hunk 1]
```diff
@@ -213,7 +218,8 @@ resource "null_resource" "bastion_packages" {
     }
     provisioner "remote-exec" {
         inline = [
-            "pip3 install ansible -q"
+            "pip3 install ansible -q",
+            "sudo ppc64_cpu --smt=2"
```
Comment (bpradipt) [inline modules/1_bastion/bastion.tf]: smt=2 or 4?

File: docs/quickstart.md [hunk 2]
```diff
@@ -67,6 +67,8 @@ Update the following variables specific to the nodes.
  * `private_key_file` : (Optional) Corresponding private key file. Default path is 'data/id_rsa'.
  * `private_key` : (Optional) The contents of an SSH key to use for the connection. Ignored if `public_key_file` is provided.
  * `public_key` : (Optional) The contents of corresponding key to use for the connection. Ignored if `public_key_file` is provided.
+ * `connection_timeout` : (Optional) Timeout in minutes for SSH connections. Default is 45 minutes.
+ * `jump_host` : (Optional) Jump server hostname/IP to be used for SSH connections. Setup password-less SSH access to the jump_host.
```
Comment (bpradipt) [inline docs/quickstart.md]: Would it make sense to clarify the password-less SSH is required from client running TF to the bastion host? 
Also TF supports key or user based access to bastion so do we really want to restrict it to password less ssh ?

File: modules/1_bastion/bastion.tf (hunk 1 above)
Comment (yussufsh) [reply modules/1_bastion/bastion.tf]: Good catch.. this change was not intended.

File: docs/quickstart.md (hunk 2 above)
Comment (yussufsh) [reply docs/quickstart.md]: Just don't want to overload the variables list. Password-less access makes life easier.

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @yussufsh

**Review by Prajyot-Parab — APPROVED** [review]

**Pull request merged**